packages/devextreme-react/src/core/configuration/react/element.ts

```diff
@@ -48,7 +49,7 @@ function getElementInfo(
     };
   }
 
-  if (reactElement.type === TemplateComp) {
+  if (reactElement.type === TemplateComp || reactElement.type === TemplateFuncComp) {
```

Comment:
That's to make it work in production (with old class components) and testing (with new functional components) in the meantime, while we are working on generators. This will go away once we finish the feature.